angular-date-pickable.js

```diff
@@ -32,15 +33,13 @@ function jbDatePickableDirective () {
     vm.isInRange = isInRange;
     vm.isSelected = isSelected;
 
-    vm.visibleMonths = 2;
-
     ////
     updateVisibleDates();
     updateSelectedDateRange(vm.startDate, vm.endDate);
 
     function updateVisibleDates () {
       vm.visibleDates = [moment(vm.visibleDate.toDate())];
-      for(var i = 1; i < vm.visibleMonths; i++) {
+      for(var i = 1; i < (vm.visibleMonths || 2); i++) {
```

Comment:
Great, but should we have a default of 1 visible month? Regardless, we should set this "magic number" as a `const`